buildSrc/src/main/groovy/io/micronaut/guides/GuideAsciidocGenerator.groovy

```diff
@@ -289,7 +276,7 @@ class GuideAsciidocGenerator {
         List<String> newLines = commonLines(file, projectDir)
 
         Optional<Integer> number = parseNumber(rawLine)
```

Comment:
is it necessary to parse the Number any more? it seems you nuked it.

Reply:
you're right, it can be removed